JamfSyncTests/DistributionPointTests.swift

```diff
@@ -736,33 +736,34 @@ final class DistributionPointTests: XCTestCase {
         XCTAssertEqual(synchronizationProgress.currentTotalSizeTransferred, 123458023)
     }
 
-    func test_transferLocal_move() throws {
-        // Given
-        let localPath = "/dst/path"
-        let fileName = "fileName.pkg"
-        let srcFile = DpFile(name: fileName, size: 123456789)
-        let synchronizationProgress = SynchronizationProgress()
-        let dstDp = MockDistributionPoint(name: "TestDstDp", fileManager: mockFileManager)
-        synchronizationProgress.printToConsole = true // So it will update before the test is over
-        synchronizationProgress.currentTotalSizeTransferred = 1234
-
-        let expectationCompleted = XCTestExpectation()
-        Task {
-            // When
-            try await dstDp.transferLocal(localPath: localPath, srcFile: srcFile, moveFrom: URL(fileURLWithPath: "/src/path/fileName.pkg"), progress: synchronizationProgress)
-
-            // For this type of DP, nothing happens, but need to make sure it returns without error.
-            expectationCompleted.fulfill()
-        }
-        wait(for: [expectationCompleted], timeout: 5)
-
-        // Then
-        XCTAssertEqual(mockFileManager.itemRemoved, URL(fileURLWithPath: "/dst/path/fileName.pkg"))
-        XCTAssertEqual(mockFileManager.srcItemMoved, URL(fileURLWithPath: "/src/path/fileName.pkg"))
-        XCTAssertEqual(mockFileManager.dstItemMoved, URL(fileURLWithPath: "/dst/path/fileName.pkg"))
-        XCTAssertEqual(synchronizationProgress.currentFileSizeTransferred, 123456789)
-        XCTAssertEqual(synchronizationProgress.currentTotalSizeTransferred, 123458023)
-   }
+    // TODO: Figure out how to mock the extension method and put this test back in
```

Comment:
Is there a ticket or GitHub issue for this to be done at a future date?

Reply:
There isn't. Not sure I want to create a GitHub issue since it's public. I'll add it to my private TODO list though.